package.json

```diff
@@ -11,34 +11,37 @@
     "test:e2e": "cypress run",
     "test:unit": "vitest"
   },
-  "devDependencies": {
+  "dependencies": {
     "@kevinmarrec/nuxt-pwa": "^0.16.2",
+    "@nuxtjs/supabase": "^0.3.1",
+    "@pinia/nuxt": "^0.4.7",
     "@types/js-cookie": "^3.0.3",
     "@types/mapbox-gl": "^2.7.10",
-    "@types/node": "^18.14.1",
+    "@vueuse/core": "^9.13.0",
+    "@vueuse/nuxt": "^9.13.0",
     "ajv": "^6.10.1",
     "csv": "^6.2.7",
     "cypress": "^12.7.0",
+    "floating-vue": "^2.0.0-beta.20",
     "got": "^9.6.0",
+    "js-cookie": "^2.2.1",
+    "kafkajs": "^2.2.4",
+    "mapbox-gl": "^2.13.0",
     "moment-range": "^4.0.2",
     "moment-timezone": "^0.5.26",
+    "normalize.css": "^8.0.1",
     "nuxt": "^3.2.2",
+    "pinia": "^2.0.32",
     "ramda": "^0.26.1",
     "sitemap": "^7.1.1",
     "slugify": "^1.3.4",
-    "vitest": "^0.29.1"
-  },
-  "dependencies": {
-    "@pinia/nuxt": "^0.4.7",
-    "@vueuse/core": "^9.13.0",
-    "@vueuse/nuxt": "^9.13.0",
-    "floating-vue": "^2.0.0-beta.20",
-    "js-cookie": "^2.2.1",
-    "mapbox-gl": "^2.13.0",
-    "normalize.css": "^8.0.1",
-    "pinia": "^2.0.32",
     "vue-virtual-scroller": "^2.0.0-beta.8"
   },
+  "devDependencies": {
+    "@types/node": "^18.14.1",
+    "supabase": "^1.41.6",
+    "vitest": "^0.29.2"
+  },
```

Comment:
Split dependencies so we only need to install prod dependencies to build the website in the netlify ci build step.